XML/TestCases/FunctionalTests.xml

```diff
@@ -1117,4 +1117,32 @@
         <Area>KDUMP</Area>
         <Tags>kdump</Tags>
     </test>
+    <test>
+        <testName>STORAGE-TAKE-REVERT-SNAPSHOT</testName>
+        <setupScript>.\TestScripts\Windows\\AddVhdxHardDisk.ps1</setupScript>
+        <testScript>SETUP-STOR-TakeRevert-Snapshot.ps1</testScript>
+        <files>.\Testscripts\Linux\utils.sh,.\Testscripts\Linux\check_traces.sh,.\Testscripts\Linux\STOR-Lis-Disk.sh</files>
+        <setupType>SingleVM</setupType>
+        <TestParameters>
+            <param>DiskType=VHD</param>
+            <param>SCSI=1,1,Dynamic</param>
+            <param>SnapshotName=ICABase</param>
```

Comment:
why is this required?

Reply:
SnapshotName=ICABase is the  given name to snapshot before the revert test